src/Controller/MediaAdminController.php

```diff
@@ -23,6 +23,15 @@ public function createAction(Request $request): Response
     {
         $this->admin->checkAccess('create');
 
+        if (null !== $request->query->get('pcode') &&  $request->isXmlHttpRequest()) {
```

Comment:
@mpoiriert if i am not wrong i guess checking also the pcode can be helpful to figure out if the modal was rendered by the collection type